db/api/v1/routes/validate.js

```diff
@@ -118,6 +118,62 @@ const extractBodyParam = (req, paramName) => req.body[paramName];
  */
 const extractParamsParam = (req, paramName) => req.params[paramName];
 
+/**
+ * Validates that a number is within the bounds of valid latitudes
+ * 
+ * @param {number} latitude - the latitude to validate
+ * @return {boolean} true if the latitude is a valid value.
+ */
+const validLatitude = (latitude) => {
+    return (latitude >= -90 && latitude <= 90);
+};
+
+/**
+ * Validates that a number is within the bounds of valid longitudes
+ *  
+ * @param {number} longitude - the longitude to validate
+ * @returns {boolean} true if the longitude is a valid value.
+ */
+const validLongitude = (longitude) => {
+    return (longitude >= -180 && longitude <= 180);
+};
+
+/**
+ * Parses a list of latitude and longitude points.
+ * @param {Array} coordinateList - a list of latitude and longitude points.
+ * @returns {ParseResult} parse success if a valid list of latitude and longitude points of at least length 3
+ */
+const parseCoordinates = (coordinateList) => {
+   
+    // There needs to be at least 3 
+    if (coordinateList.length < 3) {
+        return ParseResult.failure('Expected at least 3 points in the coordinates list');
+    }
+
+    let lat = 0;
+    let lon = 0;
+
+    const coordinates = [];
+
+    var i;
+    var point;
+    for (i = 0; i < coordinateList.length; i++) {
+        point = coordinateList[i];
+        lon = parseFloat(point.longitude);
+        lat = parseFloat(point.latitude);
+
+        if (!isNaN(lon) && validLongitude(lon) 
+            && !isNaN(lat) && validLatitude(lat)) {
+            coordinates.push({latitude: lat, longitude: lon});
+        }
+        else {
```

Comment:
Move this else onto the previous line, https://www.w3schools.com/js/js_conventions.asp